### ISSUE TITLE
Line inside chart

### ISSUE THREAD

**Issue by ipardosi16** [kranfix/flutter_radar_chart#7]
Can we add line inside the chart like chart I attached?

<img width="324" alt="Screenshot 2022-05-18 at 3 31 37 PM" src="https://user-images.githubusercontent.com/64540108/168983220-3bff3665-ce89-494f-9e2e-c31141152b1c.png">

**Comment by kranfix** [comment]
Yes, it is possible. Do you have some suggestion for the API of the widget?
